package.json

```diff
@@ -34,7 +34,7 @@
     "taskcluster-client": "0.23.4",
     "taskcluster-lib-scopes": "^0.8.8",
     "taskcluster-lib-stats": "^0.8.7",
-    "taskcluster-lib-testing": "^0.10.1",
+    "taskcluster-lib-validate": "^0.4.2",
```

Comment:
I'm a bit confused. I didn't think this was devDependency only. schema-validator-publisher wasn't. Was that a mistake?